inference-engine/src/mkldnn_plugin/nodes/mkldnn_rnn.cpp

```diff
@@ -68,6 +71,18 @@ size_t statesCount(algorithm alg) {
     }
 }
 
+bool haveCellState(algorithm alg) {
+    return alg == algorithm::vanilla_lstm;
+}
+
+const std::map<InferenceEngine::Precision, InferenceEngine::Precision> MKLDNNRNN::weightsByLayerPrec {
+    // layer precision, weights precision
+    {InferenceEngine::Precision::FP32, InferenceEngine::Precision::FP32},
+    {InferenceEngine::Precision::BF16, InferenceEngine::Precision::BF16},
+    {InferenceEngine::Precision::FP16, InferenceEngine::Precision::FP16},
+    {InferenceEngine::Precision::U8,   InferenceEngine::Precision::I8},
```

Comment:
For now the node itself support only first two combinations. I would suggest to remove others (or at least comment out).

Reply:
Done